### ISSUE TITLE
PELD client/gui localization (Chinese)

### ISSUE THREAD

**Issue by ArtificialQualia** [ArtificialQualia/PyEveLiveDPS#66]
This is to track the work for adding Chinese to the client GUI.

Current options:
- One-time release of the directly-translated strings in the code
- Proper i18n with translations in localization files, allowing easy support for other languages in the future

**Comment by ArielHeleneto** [comment]
Proper i18n with translations in localization files, allowing easy support for other languages in the future

I like this one

**Comment by ArtificialQualia** [comment]
@ArielHeleneto I agree!  This was almost completed in #76 but the dev who was working on it appears to have gone missing.  They at least provided a translation of the English phrases to Chinese so in the worst case if they do not return it should be possible to finish developing this myself.

**Comment by cnlnpjhsy** [comment]
Sorry for the late response :(
Unfortunately I have afk from EVE for a long time, but I'd still like to follow up this issue! I will find time to review #76 and try to do some localization here.
